backend/frontend/templates/frontend/user/board/js_exec_action_start_card.html

```diff
@@ -0,0 +1,5 @@
+{% include "frontend/user/board/partial_card.html" %}
+
+<script>
+  moveCardToCorrectColumn("{{card.id}}", "{{card.IN_PROGRESS}}");
```

Comment:
This script should not be needed. We have a general one that will work for moving the card to any column.